README.md

```diff
@@ -245,6 +245,22 @@ import "forge-std/console.sol";
 console.log(someValue);
 ```
 
+## Contributing
+
+See our [contributing guidelines](./CONTRIBUTING.md).
+
+## Getting Help
+
+First, see if the answer to your question can be found in [book](https://book.getfoundry.sh), or in the relevant crate.
+
+If the answer is not there:
+
+-   Join the [support Telegram](https://t.me/foundry_support) to get help, or
+-   Open a [discussion](https://github.com/foundry-rs/foundry/discussions/new) with your question, or
```

Comment:
I think this could still be useful as users generally don't know the distinction between `Foundry` / `forge-std`

Or are you in favor of deprecating either / any support portals? We haven't seen a lot of use of the Github discussions but it is a good place to find answers if we clean it up a bit

Reply:
Can be handled in a follow-up if desired cc @mds1

Reply:
Sorry I mean the discussions.